scripts/content-repo-preview/build.ts

```diff
@@ -41,35 +39,19 @@ async function main() {
   }
 
   // copy public files
-  console.log('📝 copying files in the public folder')
+  console.log('📝 Copying files from "./public" to "../"')
   execFileSync('cp', ['-R', './public', '../'])
 
-  /**
-   * Remove dirs in `src/pages` which are not associated with the product
-   */
-  const pagesDir = path.join(cwd, 'pages')
-
-  const pagesDirs = (
-    await fs.promises.readdir(pagesDir, { withFileTypes: true })
-  ).filter((ent) => ent.isDirectory())
-
-  for (const dir of pagesDirs) {
-    if (!dir.name.includes(repo) && dir.name !== 'home') {
-      console.log(`🧹 removing pages for ${dir.name}`)
-      await fs.promises.rm(path.join(pagesDir, dir.name), {
-        recursive: true,
-      })
-    }
-  }
-
   /** Install deps */
   console.log('📦 Installing dependencies')
   execFileSync('npm', ['install', '--production=false'], { stdio: 'inherit' })
 
   /** Build */
+  console.log('🏗️  Building project')
   execFileSync('npm', ['run', 'build'], { stdio: 'inherit' })
 
   // Put node_modules into .next/cache so we can retrieve them on subsequent builds
+  console.log('🐿 Copying "node_modules" to ".next/cache"')
```

Comment:
Excellent emoji choice 😄

Reply:
I honestly do not remember if GitHub Copilot wrote this or not 😂 ... but it makes perfect sense to me!